src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -231,26 +232,26 @@ public void clickSectionCheckBoxInSectionLevel(int instrNum, int sectionLevelNum
     }
     
     public void clickViewStudentCheckBoxInSectionLevel(int instrNum, int sectionLevelNum) {
-        String cssSelector = "#tuneSectionPermissionsDiv" + sectionLevelNum + "ForInstructor" + instrNum
-                + " > div > div.panel-body > div.col-sm-6.border-right-gray > input[type=\"checkbox\"]:nth-child(1)";
+        String cssSelector = "input[type=\"checkbox\"][name=\"canviewstudentinsectionsectiongroup"
+                             + sectionLevelNum + "\"]";
```

Comment:
Yep, works even without `instrNum` as only one instructor's checkboxes are visible at any time. Will change this.